app/src/main/AndroidManifest.xml

```diff
@@ -96,6 +97,16 @@
             android:name=".settings.SettingsActivity"
             android:parentActivityName=".webview.WebViewActivity" />
 
+        <activity android:name=".nfc.NfcActivity"
+            android:parentActivityName=".webview.WebViewActivity">
+            <intent-filter>
+                <action android:name="android.nfc.action.NDEF_DISCOVERED" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <data android:scheme="https"
+                    android:host="www.home-assistant.io"
+                    android:pathPrefix="/android/nfc/" />
```

Comment:
Let's settle on a link format right now and implement it correctly right away.

CC @home-assistant/ios-developers

Reply:
How about we split the difference and do `/nfc/*`? I thought about adding mobile as a prefix but there's no reason to bake that into NFC.

Reply:
I wonder if it should be /tag/ because NFC is an implementation detail.

Reply:
It's potentially an issue but we could also be stuck trying to contort some future technology into today's limitations.

The current parallel may be iBeacon being seen (proximity in or out). I wonder if we can extend the tag read event were defining below to account for them. But they aren't really URL-based.

Reply:
@zacwest yes those beacons are not url based and android does not natively support them. So from my understanding we would have to include a third party library that does background scanning using bluetooth. However there are some restrictions on newer devices so that the detection might be delayed by about 300s (https://altbeacon.github.io/android-beacon-library/detection_times.html). However I've never used them in production.. maybe there are easier and better ways to use them however I don't have an iBeacon at hand to test with.

As iBeacons (can?) contain a uuid, it might be possible to use them in the same way as the nfc tags. However that'd require some more research and testing.

Reply:
I'm mainly thinking about RFID cards using non-phone apps as readers. https://github.com/maddox/magic-cards

Reply:
If we want to tack a URL onto an identifier (which is what this path is) I'd imagine we're painting ourselves less into a corner. Requiring a URL format for RFID might be overkill (or exceed maximum capacity, etc.) is pushing today's limitations on tomorrow's technology. I'd imagine we say `home-assistant.io/nfc` is more of a namespace than a standard.

However, the RFID point about the _event_ being fired is very convincing, and (comment below) I think we should take the `nfc` out of it.

Reply:
Sounds good

Reply:
I thought about this more and I think I am wrong. For example, a QR code shouldn't require a new implementation for the apps to handle it. What we have are a class of things (URL-based tags) and other things (raw/limited tags). The latter will require unique approaches by their nature.

So new proposal: `https://www.home-assistant.io/tag/<id>` for all URL-based tags. How the app get launched for a URL matters less -- just that it did.